src/Mod/Fem/feminout/writeFenicsXML.py

```diff
@@ -68,17 +68,17 @@ def write_fenics_mesh_xml(fem_mesh_obj, outputfile):
         "hexahedron": 8
     }
 
-    Console.Message("Converting " + fem_mesh_obj.Label + " to fenics XML File\n")
-    Console.Message("Dimension of mesh: %d\n" % (get_FemMeshObjectDimension(fem_mesh_obj),))
+    Console.PrintMessage(u"Converting " + fem_mesh_obj.Label + u" to fenics XML File\n")
```

Comment:
Why do you need to prepend with `u"string"`? I don't think this is necessary. Maybe it was used before, with Python 2, but it's not required anymore.

Reply:
@vocx-fc thanks for pointing this out. We were just not sure about that. What do we have to do to maintain backwards compatibility?

Reply:
Mmm. Not sure, but maybe nothing? I mean, you should only add the `u""` if you really need it. Do you notice any difference if you don't add the `u""`? If not, then don't add it.

Reply:
In python3 (which can now be considered FreeCAD default, as we are slowly abandonning py2 support), all strings are unicode. So u"Something" is fully equal to "Something". I would also remove the u I guess